### ISSUE TITLE
[MU4 Issue] Col legno playback on viola is out of tune

### ISSUE THREAD

**Issue by Attackoh** [musescore/MuseScore#15568]
**Describe the bug**
When playing G repeatedly on viola, the playback will sound like the notes F E C B and loop identically in that exact order. Other notes have a lesser degree of this problem.
**To Reproduce**
1. Create viola
2. Add col legno from palettes 
4. Add many Gs
5. Playback

**Expected behavior**
Pitch should be more in tune and should have less variance. It should not loop identically each time as to not sound unrealistic.
**Screenshots**
![image](https://user-images.githubusercontent.com/120669196/209446619-a3c4065d-7fba-4f3b-a093-d88f1694152f.png)

**Platform information**
 - OS: Windows

**Additional context**

**Comment by cbjeukendrup** [comment]
Yeah, I can confirm, with solo viola, and actually also section violas. 

Also, when you play a long sequence of 16th col legno notes, the col legno "attack noise" starts sounding like it is not related to the pitches themselves. In other words, it doesn't sound like both are caused by the same physical event on one instrument.
